sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -177,9 +176,8 @@ __spirv_VectorInsertDynamic(__spv::__spirv_JointMatrixINTEL<T, R, C, L, S, U> *,
 
 template <typename T>
 extern __DPCPP_SYCL_EXTERNAL void __spirv_CooperativeMatrixPrefetchINTEL(
-    T *Ptr, std::size_t coordX, std::size_t coordY, std::size_t NumRows,
-    std::size_t NumCols, unsigned int CacheLevel, __spv::MatrixLayout Layout,
-    std::size_t Stride);
+    T *Ptr, int32_t coordX, int32_t coordY, uint32_t NumRows, uint32_t NumCols,
```

Comment:
coordX and coordY were removed from the spec. Please make sure you remove them in SPIRV as well.

Reply:
Lets update it step by step and keep coordinates in this PR :) Prefetch requires some update starting from the translator and ending up here.